cli/azd/cmd/provision.go

```diff
@@ -125,6 +129,23 @@ func (p *provisionAction) Run(ctx context.Context) (*actions.ActionResult, error
 		defaultTitleNote = "This is a preview. No changes will be applied to your Azure resources."
 	}
 
+	// Get Subscription to Display in Command Title Note
+	subscription, subErr := p.subService.GetSubscription(ctx, p.env.GetSubscriptionId(), p.env.GetTenantId())
```

Comment:
This might be an expensive call.  Wondering if we already have that context or if we can cache it in .env as well once we get it.

Reply:
I checked all the env variables and tried multiple times before digging deep into the accounts.
You only store the id to the env variable.
There is another function in the Subscription Manager (GetSubscriptions) but I saw that it checks for an extra thing which is the tenant so, I thought using this as it doesn't check for the tenant would be better.

Correct me if I'm wrong!

Reply:
done